CredScanSuppressions.json

```diff
@@ -11,6 +11,7 @@
     },
     {
       "file": [
+        "src\\azure-cli\\azure\\cli\\command_modules\\appconfig\\tests\\latest\\recordings\\test_appconfig_to_appconfig_import_export.yaml",
```

Comment:
@haroldrandom  please help look at this credential scan issue.

Reply:
![image](https://user-images.githubusercontent.com/14357159/72200025-3ef5ab80-347f-11ea-80c4-720114c3d5c5.png)
![image](https://user-images.githubusercontent.com/14357159/72200039-6482b500-347f-11ea-9b5e-fcc4bee786e8.png)
This 2 response bodies are being recognized as secrets.